xinference/model/llm/pytorch/utils.py

```diff
@@ -352,7 +352,7 @@ def generate_stream_falcon(
     streamer = TextIteratorStreamer(tokenizer, skip_prompt=True, **decode_config)
 
     generation_config = GenerationConfig(
-        max_new_tokens=max_new_tokens,
+        max_tokens=max_tokens,
```

Comment:
The GenerationConfig defined in transformer is `max_new_tokens`. We can unify `max_tokens` at xinference level, but when using `generate` method of transformer, only `max_new_tokens` can be used.
It is recommended to still use `max_new_tokens` in pytorch/utils.py:
```
max_new_tokens = int(generate_config. get("max_tokens", 256))
```

https://huggingface.co/docs/transformers/main/en/main_classes/text_generation#transformers.GenerationConfig

This line must be:
```
generation_config = GenerationConfig(
        max_new_tokens=...,
        ...,
)
```